reload_app/app.py

```diff
@@ -11,16 +12,16 @@
 from ua_parser import user_agent_parser
 from uuid import uuid1
 
-
 from .events import VALID_EVENTS
 from .metrics import VALID_METRICS, VALID_GLOBAL_TAGS
 from .metrics.dogstatsd import DogStatsdMetrics
-from .raven_client import client
 from .router import Router
 from .worker import BigQueryWorker
 from .utils import format_datetime, ip_from_request
 from .geo import geo_by_addr
 
+sentry_sdk.init(os.environ.get("SENTRY_DSN"))
```

Comment:
```suggestion
sentry_sdk.init()
```

The SDK [gets the DSN from the environment](https://github.com/getsentry/sentry-python/blob/cad2f65316bab4ee5792b1b788c32c57293eea5e/sentry_sdk/client.py#L61-L62) by default, if you prefer to omit the argument.